storage/remote/otlptranslator/prometheus/normalize_name_test.go

```diff
@@ -85,36 +85,20 @@ func TestEmpty(t *testing.T) {
 func TestAllowUTF8(t *testing.T) {
 	t.Run("allow UTF8", func(t *testing.T) {
 		require.Equal(t, "unsupported.metric.temperature_°F", normalizeName(createGauge("unsupported.metric.temperature", "°F"), "", true))
-		require.Equal(t, "unsupported.metric.weird_+=.:,!* & #", normalizeName(createGauge("unsupported.metric.weird", "+=.:,!* & #"), "", true))
+		require.Equal(t, "unsupported.metric.+weird_+=.:,!* & #", normalizeName(createGauge("unsupported.metric.+weird", "+=.:,!* & #"), "", true))
```

Comment:
@aknuds1, is this the test you wanted to see?

Reply:
Yep, that is the one.